xrmocap/service/smpl_stream_service.py

```diff
@@ -199,6 +204,28 @@ def upload_smpl_data(self, data: dict) -> dict:
         file_path = os.path.join(self.work_dir, f'{uuid_str}_{file_name}.npz')
         with open(file_path, 'wb') as file:
             file.write(file_data)
+        data_type = self.data_converter.get_data_type(file_path)
+        # organize the input data as the smpl data
+        if data_type is SMPLDataTypeEnum.AMASS:
+            self.logger.info('Received AMASS data, converting to SMPL(X) data')
+            data = self.data_converter.from_amass(file_path)
+            data.dump(file_path)
+        elif data_type is SMPLDataTypeEnum.HUMANDATA:
+            self.logger.info('Received HumanData, converting to SMPL(X) data')
+            data = self.data_converter.from_humandata(file_path)
```

Comment:
Easier to find out what type the return value is.
```suggestion
            smpl_data = self.data_converter.from_humandata(file_path)
```
What if `self.data_converter` returns None?